docs/etcd_clusters.md

```diff
@@ -0,0 +1,60 @@
+# Manage etcd clusters
+
+Mayu contains an etcd discovery to setup and manage your etcd clusters. During startup you can configure if you like to use an external etcd discovery or Mayu itself.
```

Comment:
Ah, I see. 👍 . 

May I recommend a single NOTICE in the text below then, to make that more explicit? I would bet others will stumble upon this too.

> Use Mayus discovery (Note that this does not use `--etcd-discovery`, so Mayu uses it's own included etcd-discovery server):